cypress/integration/Crosscutting/PublicUseTest/index.js

```diff
@@ -0,0 +1,31 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('user is navigating to {string}', (a) => {
+    cy.visit(a);
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message 'list not defined'
+        // and don't want to fail the test so we return false
+        if (err.message.includes('Cannot read property')) {
+            return false
+        }
+        // we still want to ensure there are no other unexpected
+        // errors, so we let them fail the test
+    })
+});
+
+Then('public use text is displayed', () => {
+    cy.get('div.public-use').should('be.visible');
+});
+
+And('the public use text contains {string}', (title) => {
+    cy.get('div.public-use').should('include.text', title);
+});
+
+Then('public use text is not displayed', () => {
+    cy.get('div.public-use').should('not.exist');
+});
+
+Then('the length of public use text is within correct range', () => {
+    cy.get('div.public-use').should('have.length.lessThan', 500);
```

Comment:
And probably add moreThan a certain number too ? ( like length 1 would mean that something is not right )

Reply:
fixed this